CONTRIBUTING.md

```diff
@@ -106,3 +106,7 @@ $ open coverage/lcov-report/index.html
 # Linux
 $ xdg-open coverage/lcov-report/index.html
 ```
+
+### Submitting Pull Requests
+
+If you've cloned the project directly and are working on a feature branch, you'll need write permissions to be able to push that branch to the remote to open a pull request.
```

Comment:
I don't see a reason to share write permission with contributors. Fork is only right way.